Makefile

```diff
@@ -17,6 +17,13 @@ else
 	VERSION := $(TAG)-dirty
 endif
 
+pvpath := $(shell which pv 2>/dev/null || true)
+ifeq ($(pvpath),)
+	PV :=
+else
+	PV := | $(pvpath)
+endif
+
```

Comment:
[This](https://stackoverflow.com/a/677212) stack overflow answer suggests to use `command -v` instead. What do you think?

Reply:
That works. Updated.